nullaway/src/test/java/com/uber/nullaway/NullAwayJSpecifyGenericsTests.java

```diff
@@ -155,6 +155,7 @@ public void returnTypeParamInstantiation() {
             "  static class NullableTypeParam<E extends @Nullable Object> {}",
             "  // BUG: Diagnostic contains: Generic type parameter",
             "  static NonNullTypeParam<@Nullable String> testBadNonNull() {",
+            "    // BUG: Diagnostic contains: Cannot return",
             "    return new NonNullTypeParam<String>();",
```

Comment:
Let's remove the return error here to keep the test clean:
```suggestion
            "    return new NonNullTypeParam<@Nullable String>();",
```